src/main/java/org/prgrms/cream/domain/deal/service/SellingService.java

```diff
@@ -0,0 +1,66 @@
+package org.prgrms.cream.domain.deal.service;
+
+import java.util.List;
+import org.prgrms.cream.domain.deal.domain.BuyingBid;
+import org.prgrms.cream.domain.deal.domain.Deal;
+import org.prgrms.cream.domain.deal.dto.BuyRequest;
+import org.prgrms.cream.domain.deal.dto.DealResponse;
+import org.prgrms.cream.domain.deal.model.DealStatus;
+import org.prgrms.cream.domain.product.domain.ProductOption;
+import org.prgrms.cream.domain.product.service.ProductService;
+import org.prgrms.cream.domain.user.service.UserService;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class SellingService {
+
+	private static final int FIRST_BID = 0;
+	private static final int SECOND_BID = 1;
+	private static final int ZERO = 0;
+	private static final int ONLY_ONE_BID = 1;
+	private static final int ALL_BID = 2;
+	private final ProductService productService;
+	private final UserService userService;
```

Comment:
```suggestion
	private static final int ALL_BID = 2;
	
	private final ProductService productService;
	private final UserService userService;
```

개행하면 구분하기 좋을 것 같습니다!

Reply:
e05f0dc : 개행 추가했습니다!! 확실히 이전보다 구분이 잘되는 느낌이에요!!